test/Microsoft.Extensions.Logging.Test/EventLogLoggerTest.cs

```diff
@@ -149,9 +148,7 @@ public void MessageWithinMaxSize_WritesFullMessage(int messageSize)
             }
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux)]
-        [OSSkipCondition(OperatingSystems.MacOSX)]
+        [Theory]
```

Comment:
This type is `#ifdef`ed for `NET451`. So the conditional here wasn't required. Removing it lets us update testing independent of this PR.